gobblin-core/src/main/java/gobblin/writer/AvroHdfsDataWriter.java

```diff
@@ -167,6 +168,24 @@ public void commit()
     if (!this.fs.rename(this.stagingFile, this.outputFile)) {
       throw new IOException("Failed to commit data from " + this.stagingFile + " to " + this.outputFile);
     }
+
```

Comment:
I still think it should be a utility method. A better place to put it is `WriterUtils`.